docs/components/board/_index.md

```diff
@@ -0,0 +1,1565 @@
+---
+title: "Board Component"
+linkTitle: "Board"
+weight: 20
+type: "docs"
+no_list: true
+description: "The signal wire hub of a robot, with GPIO pins for transmitting signals between the robot's computer and its other components."
+tags: ["board", "components"]
+icon: "/components/img/components/board.svg"
+# SMEs: Gautham, Rand
+---
+
+A *board* is the signal wire hub of a robot that provides access to [general purpose input/output (GPIO) pins](https://www.howtogeek.com/787928/what-is-gpio/).
+
+If your board has a computer that is capable of running `viam-server`, or is connected to one, it can act not only as the signal wire hub for your robot, but also as the software hub.
+
+Configure a board component on your robot to control and read from the other [components](/components) of the robot, signaling through the GPIO pins on the board as overseen by a computer running `viam-server`.
+
+#### What are GPIO pins?
+
+GPIO pins are pins on the motherboard of a computer that can receive electrical signals.
+You can control the flow of electricity to the pins to change their state between "high" (active, >0V) and "low" (inactive, 0V), and wire them to send [digital signals](https://en.wikipedia.org/wiki/Digital_signal) to and from other hardware.
+
+Many GPIO implementations also support [Pulse Width Modulation (PWM)](https://en.wikipedia.org/wiki/Pulse-width_modulation), or can be used to communicate with protocols like [I<sup>2</sup>C](#i2cs) or [SPI](#spis).
+
+#### What can I use as my board?
+
+Generally, you should use a single-board computer (SBC) with GPIO pins or a computer outfitted with a GPIO peripheral.
+
+This way, overseen by a computer running `viam-server`, the GPIO pins on your board can receive signals from and send signals to the other [components](/components/) of your robot.
+
+**Single-Board Computer with GPIO Pins:**
+
+This refers to boards like the [Raspberry Pi](/installation/prepare/rpi-setup/), [BeagleBone](/installation/prepare/beaglebone-setup/), and [Jetson](/installation/prepare/jetson-nano-setup/).
+
+These are all small computing devices outfitted with GPIO pins that are capable of advanced computation, including running `viam-server`.
+
+{{% alert title="Note" color="note" %}}
+If you want to use the GPIO pins on your SBC to control your robot, the board itself must run `viam-server`.
+The GPIO pins on SBCs are generally not accessible to external computers.
+{{% /alert %}}
+
+**GPIO Peripheral and Computer**:
+
+A device with GPIO pins, like the [numato](numato) model of board, can still act as the signal wire hub of your robot.
```

Comment:
```suggestion
A device with GPIO pins but no CPU, like a [Numato GPIO peripheral](https://numato.com/product-category/automation/gpio-modules/), can still act as the signal wire hub of your robot.
```
Changing the link is opt but I’d recommend adding some qualifier because if it just says with GPIO pins that could apply to any of the other boards.